packages/OptionButton/src/option-button.scss

```diff
@@ -0,0 +1,158 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-option-button-border-color: #{tokens.$grey-400};
+  --ids-option-button-border-radius: #{tokens.$border-radius-sm};
+  --ids-option-button-font-family: #{tokens.$primary-font-family};
+
+  /* Focus */
+  --ids-option-button-focus-outline-color: #{tokens.$electric-blue-600};
```

Comment:
`--ids-option-button-focus-outline-color` is not used

Reply:
right, that was changed so I can remove that.